recipe/patches/fix_openjpeg.diff

```diff
@@ -0,0 +1,46 @@
+--- frmts/openjpeg/CMakeLists.txt 	2024-07-02 10:40:13.286433817 -0500
```

Comment:
you probably don't need that patch, but instead build libgdal-core with the openjpeg driver available but with -DGDAL_USE_OPENJPEG=OFF
Cf text below https://gdal.org/development/building_from_source.html#cmdoption-arg-OGR_REGISTER_DRIVER_-driver_name-_FOR_LATER_PLUGIN-BOOL

Reply:
Yes, it's not used anymore. I've removed the patch from the git repo now.